src/09-tool-patterns/03-vanilla-tool-call.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Demonstrates that tools defined via Vercel AI SDK and
+ * can easily be called via the vendor's own API client and
+ * need not be locked into the Vercel abstractions.
```

Comment:
Once concern when reaching for these abstractions is how heavy and sticky they are, in terms of lock-in, escape hatches etc. So this example was just to prove to myself that the tool abstraction works just fine if we need to break out and supply it directly to one of the vendor's API clients.